src/dex/fluid-dex/fluid-dex-liquidity-proxy.ts

```diff
@@ -51,30 +54,21 @@ export class FluidDexEventPool extends StatefulEventSubscriber<FluidDexPoolState
    */
   async handleOperate(
     event: any,
-    state: DeepReadonly<FluidDexPoolState>,
+    state: DeepReadonly<FluidDexLiquidityProxyState>,
     log: Readonly<Log>,
-  ): Promise<DeepReadonly<FluidDexPoolState> | null> {
-    if (!(event.args.user in [this.pool])) {
-      return null;
-    }
+  ): Promise<DeepReadonly<FluidDexLiquidityProxyState> | null> {
     const resolverContract = new Contract(
       this.commonAddresses.resolver,
       ResolverABI,
       this.dexHelper.provider,
     );
```

Comment:
We can create this contract only once in constructor

Reply:
@KanievskyiDanylo solved